Pinta.Core/Managers/WorkspaceManager.cs

```diff
@@ -351,7 +353,27 @@ private void ShowOpenFileErrorDialog (Window parent, string filename, string pri
 			PintaCore.Chrome.ShowErrorDialog(parent, message, details);
 		}
 
-#region Public Events
+		private void ShowUnsupportedFormatDialog (Window parent, string filename, string primaryText, string details)
+		{
+			string markup = "<span weight=\"bold\" size=\"larger\">{0}</span>\n\n{1}";
+
+			string secondaryText = string.Format(Catalog.GetString("Could not open file: {0}"), filename);
+			secondaryText += $"{Environment.NewLine}{Environment.NewLine}Pinta supports the following file formats:{Environment.NewLine}";
+			var extensions = from format in PintaCore.System.ImageFormats.Formats
+						  from extension in format.Extensions
```

Comment:
Maybe this should also check whether an exporter exists for the format? I'm not sure if there are any write-only formats, though.

Reply:
I checked available formats for existing of exporter and found that around the half of the formats are available only for import. So I see two possible solutions:
1. Show formats only available in both ways (Import/Export).
2. Show import and export formats separately. More sophisticated way for a user, I'm not sure is it worth it.
I like more simple way more, though.
For now, I changed LINQ with additional where statement:
`where format.Exporter != null`
But can change it without any problem.

Reply:
Thanks! Should it be the other way around, though? (checking `format.Importer != null`) since this error is show when a file fails to import.

Reply:
Done.